src/renderer/components/dashboard/Price.vue

```diff
@@ -177,11 +177,7 @@ export default {
                     tickMarkLength: 20,
                   },
                   ticks: {
-                    callback(label, index) {
-                      if (index % 2 !== 0) {
```

Comment:
Since this wasn't actually doing anything to reduce the number of tool tips; removing it seems reasonable.